src/pages/WorkProgressSummary.tsx

```diff
@@ -137,33 +216,45 @@ export const summaryStatusSort = (rowAStatus: WorkStatus, rowBStatus: WorkStatus
 export default WorkProgressSummary;
 const columns: Column<WorkProgressSummaryTableEntry>[] = [
   {
-    Header: 'Work Type',
+    Header: () => <p className="font-bold text-center inline-block">Work Type</p>,
     accessor: 'workType',
     sortType: (rowA, rowB) => {
       return alphaNumericSortDefault(rowA.original.workType, rowB.original.workType);
     }
   },
   {
-    Header: 'Status',
+    Header: () => <p className="font-bold text-center">Status</p>,
```

Comment:
Do we actually needs these style? Even otherwise it will be displayed as bold ,I think

Reply:
Yeah the styling doesnt change much visually and havent heard back from minal so will leave styling for the time being